crates/math/src/fft/polynomial.rs

```diff
@@ -27,7 +27,9 @@ impl<E: IsField> Polynomial<FieldElement<E>> {
     ) -> Result<Vec<FieldElement<E>>, FFTError> {
         let domain_size = domain_size.unwrap_or(0);
         let len = core::cmp::max(poly.coeff_len(), domain_size).next_power_of_two() * blowup_factor;
```

Comment:
this is super weird, when choosing a blowup factor that is not a power of 2 it fails. Don't you think this should be changed to `log_blowup_factor` and then
```suggestion
        let len = core::cmp::max(poly.coeff_len(), domain_size).next_power_of_two() * (1 << blowup_factor);
```

Reply:
We could, but this would imply changing several parts in the stark protocol

Reply:
don't you think it would be a useful change ?

Reply:
Ok then maybe for another PR